pvtrace/Trace.py

```diff
@@ -724,8 +761,12 @@ def container(self, photon):
             return containers[min_index]
 
 class Tracer(object):
-    """An object that will fire multiple photons through the scene."""
-    def __init__(self, scene=None, source=None, throws=1, steps=50, seed=None, use_visualiser=True, show_log=False, background=(0.957, 0.957, 1), ambient=0.5, database_file="pvtracedb.sql"):
+    """An object that will fire multiple photons through the scene.
+    
+    parallel-pvtrace edit: Even when selected False, the visualiser is still communicated with, confusing parallelpython. Hence lots of try/except cases to overlook the comm issues.
+    
+    """
+    def __init__(self, scene=None, source=None, throws=1, steps=2000, seed=None, use_visualiser=False, show_log=False, background=(0.957, 0.957, 1), ambient=0.5, database_file="/tmp/pvtracedb.sql", combined_database_file = "/tmp/pvtracedbCombined.sql", parts = 1):
```

Comment:
Again hard coded paths in the keyword arguments should not be committed if we are support Windows/macOS/Linux etc..